docs/index.md

```diff
@@ -62,14 +62,29 @@ In order to run a local demo based on the SMPKit:
           }
         ]
         ```
+5. Export the necessary environment variables.
 
-5. Build the service deployment.
+    ```bash
+    # set the Goerli RPC endpoint for all the agents in the service
+    export SERVICE_AUTONOMOUS_FUND_RPC_0=YOUR_RPC_ENDPOINT
+    export SERVICE_AUTONOMOUS_FUND_RPC_1=YOUR_RPC_ENDPOINT
+    export SERVICE_AUTONOMOUS_FUND_RPC_2=YOUR_RPC_ENDPOINT
+    export SERVICE_AUTONOMOUS_FUND_RPC_3=YOUR_RPC_ENDPOINT
+    
+   # set all participants
+   export ALL_PARTICIPANTS='[["0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65","0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc","0x976EA74026E726554dB657fA54763abd0C3a0aa9","0x14dC79964da2C08b23698B3D3cc7Ca32193d9955"]]'
```

Comment:
These should be the user's keys. Should we not add something similar to the above?
```suggestion

   export ALL_PARTICIPANTS='[["your_key0","your_key1","your_key2","your_key3"]]'
```

Reply:
they are the keys from above, perhaps we make it clear that they can replace it with those and run it. Or just add a comment that these are the keys from above, and in reality you would need to replace this with your own.

Reply:
These are the addresses in the keys.json file of the service running in the example, aren't they?

Reply:
Yes